docker-compose.override.yml.dist

```
@@ -0,0 +1,6 @@
+services:
+  prestashop-git:
+    environment:
+      #BLACKFIRE_ENABLE: 1
+      #BLACKFIRE_SERVER_ID: "your_server_id"
+      #BLACKFIRE_SERVER_TOKEN: "your_server_token"
```

```suggestion
      BLACKFIRE_ENABLE: 1
      BLACKFIRE_SERVER_ID: "your_server_id"
      BLACKFIRE_SERVER_TOKEN: "your_server_token"
```

It's already in a dist file, no need to comment since you have to remove the extension for this to be taken into account

---

true...  but I thought people could rename this file and use it for other purposes...

Then these blackfire values would not make sense, I find it less dangerous to be sure that installing blackfire is 100% intentional, what do you think ?

---

The file must be a model allowing a basic config, free for anyone wanting to have their own override to remove unwanted services, like phpmyadmin on develop https://github.com/PrestaShop/PrestaShop/blob/8457cb32fe3badcb15993f18eedd318e6dbbc33f /docker-compose.override.yml.dist#L12

---

yes, but the uncommented values here are values that won't install additional tools in the containers, they configure things that will be installed no matter what

I find it safer to comment the parts that will add additional stuff

that being said I might switch to a blackfire image

---

it's true that we don't comment phpmyadmin in this file on develop though

---

You can leave the variables uncommented with the value 0 (disabled by default)

---

Something like
```suggestion
      # Disabled by default, set 1 here to enable Blackfire Don't forget to specify the server settings as well
      BLACKFIRE_ENABLE: 0
      BLACKFIRE_SERVER_ID: "your_server_id"
      BLACKFIRE_SERVER_TOKEN: "your_server_token"
```